scalatest/src/main/scala/org/scalatest/tools/Framework.scala

```diff
@@ -737,12 +739,14 @@ class Framework extends SbtFramework {
         paths.exists(path => td.fullyQualifiedName.startsWith(path) && td.fullyQualifiedName.substring(path.length).lastIndexOf('.') <= 0)
       }
       
-    def tasks(taskDefs: Array[TaskDef]): Array[Task] = 
-      for { 
+    def tasks(_taskDefs: Array[TaskDef | Null] | Null): Array[Task | Null] = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.

Reply:
**Overridden java method returns array**
Overridden java method returns an array. Java arrays have type `Array[T | Null]`. Returned array must thereby be converted to have type `Array[T | Null]`.